backend/src/main/kotlin/org/loculus/backend/model/ReleasedDataModel.kt

```diff
@@ -42,6 +44,16 @@ class ReleasedDataModel(
             .map { computeAdditionalMetadataFields(it, latestVersions, latestRevocationVersions) }
     }
 
+    @Transactional(readOnly = true)
+    open fun getLastDatabaseWriteETag(): String {
+        val lastUpdateTime = UpdateTrackerTable.selectAll()
+            .mapNotNull { it[UpdateTrackerTable.lastTimeUpdatedDbColumn] }
```

Comment:
Isn't this missing a `where table_name in (some subset of tables)`?

Reply:
We only track the tables that matter at the moment, see the migration - we could later subset this, that's true.

Reply:
I'd do it now. If someone else later adds other tables to the tracking, adapting this here is very likely to be forgotten.

Reply:
@anna-parker could you subset the tables her? Fabian makes a good point that it doesn't harm to do the subsetting explicitly here

Reply:
@corneliusroemer I just added this option - I was now actually wondering if for get-released-data we can actually already take a subset, I guess we don't really care about group-related changes so we could alert on changes in tables: 
['sequence_entries', 'sequence_entries_preprocessed_data', 'external_metadata', 'current_processing_pipeline', 'metadata_upload_aux_table', 'sequence_upload_aux_table']

Reply:
at the moment we still return new data when the groups_table and user_groups_table change

Reply:
I added this now - please check its ok.